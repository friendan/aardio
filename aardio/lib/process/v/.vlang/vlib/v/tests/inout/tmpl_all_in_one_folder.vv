fn abc() string {
	content := 'some string'
	return $tmpl('file.md')
}

fn main() {
	print(abc())
}
