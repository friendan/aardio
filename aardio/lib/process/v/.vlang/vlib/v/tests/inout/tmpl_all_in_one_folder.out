my header

This is the main content:
-------------------------
some string
-------------------------

my footer
